server/src/domain/person/person.service.ts

```diff
@@ -117,4 +117,53 @@ export class PersonService {
 
     return true;
   }
+
+  async mergePerson(authUser: AuthUserDto, primaryPersonId: string, dto: MergePersonDto) {
+    for (const id of dto.ids) {
+      const person = await this.repository.getById(authUser.id, id);
+      if (!person) {
+        throw new BadRequestException();
+      }
+    }
+
+    const primaryPerson = await this.repository.getById(authUser.id, primaryPersonId);
+    if (!primaryPerson) {
+      throw new BadRequestException();
+    }
+
+    const mergeIds = dto.ids;
+    for (const mergePersonId of mergeIds) {
```

Comment:
Just a nit, but these aren't consistent. Either use `mergePersonIds` _or_ just use `const mergeId of mergeIds` (what I'd prefer). Consistency improves readability IMO.